.github/workflows/publish.yml

```diff
@@ -0,0 +1,18 @@
+name: publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
For backward compatibility, shall we use an old version of Node.js?

Reply:
For running `npm publish` I don't think the version of Node.js would affect backward compatibility.